test/fail_compilation/diag9191.d

```diff
@@ -14,11 +14,13 @@ class B1 { void aa(); }
 class C1 : B1, I1
 {
     override void aaa();
+    void a() {}
```

Comment:
Otherwise an additional error will be reported. The test is about error messages for methods annotated with `override` which don't override any methods. Not sure what the interface is doing there to being with. Perhaps I should remove the interface instead?

The reason why this hasn't been triggered before is that since it generates an error the compilation won't continue to the code gen phase (where interface implementations are checked).

Reply:
It's probably there to verifiy that the compiler suggests `aa` instead of `a` on error. Maybe add your error message to the expected output?